pom.xml

```diff
@@ -733,12 +733,12 @@
                     <limit>
                       <counter>LINE</counter>
                       <value>COVEREDRATIO</value>
-                      <minimum>0.94</minimum>
+                      <minimum>0.96</minimum>
                     </limit>
                     <limit>
                       <counter>BRANCH</counter>
                       <value>COVEREDRATIO</value>
-                      <minimum>0.85</minimum>
+                      <minimum>0.84</minimum>
```

Comment:
can we increase this one again to 0.85?

Reply:
Yes, when sonar will be upgarded, I will remove the if conditions which were introduced because of the older version, the branch coverage will increase then.